pulp_rpm/app/viewsets.py

```diff
@@ -123,6 +123,124 @@ class RpmRepositoryViewSet(RepositoryViewSet, ModifyRepositoryActionMixin):
     endpoint_name = "rpm"
     queryset = RpmRepository.objects.exclude(user_hidden=True)
     serializer_class = RpmRepositorySerializer
+    queryset_filtering_required_permission = "rpm.view_rpmrepository"
+
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list"],
+                "principal": ["authenticated"],
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.view_rpmrepository",
+            },
+            {
+                "action": ["retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_or_obj_perms:rpm.view_rpmrepository",
+            },
+            {
+                "action": ["create"],
+                "principal": "authenticated",
+                "effect": "allow",
+                "condition": "has_model_perms:rpm.add_rpmrepository",
```

Comment:
you should also consider view permission on the remote which can be used when repo is created